src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/validation/ApplicationValidation.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.adoption.adoptioncase.validation;
+
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.Application;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseData;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static uk.gov.hmcts.reform.adoption.adoptioncase.validation.ValidationUtil.flattenLists;
+// import static uk.gov.hmcts.reform.adoption.adoptioncase.validation.ValidationUtil.validateApplicant2BasicCase;
+import static uk.gov.hmcts.reform.adoption.adoptioncase.validation.ValidationUtil.validateBasicCase;
+
+public final class ApplicationValidation {
+
+    private ApplicationValidation() {
+
+    }
+
+    public static List<String> validateReadyForPayment(CaseData caseData) {
+        return validateBasicCase(caseData);
```

Comment:
Please check once ApplicationValidation commented methods body

Reply:
Yes this will be added in application submission story